function-reference/build-functions/text.xml

```diff
@@ -24,7 +24,27 @@ during the unpack and compile stages.
       <c>unpack archives</c>
     </ti>
     <ti>
-      Unpack the specified archives.
+      <p>
+        Unpack the specified archives. Recognised formats are:
```

Comment:
Aren't we using AmE spelling in devmanual, i.e. `recognized`?

Reply:
https://devmanual.gentoo.org/appendices/contributing/index.html#style-guidelines
> This document is in British English. Submissions in other kinds of English are welcome, but they may have their spelling corrected. 

Anyway, Macmillan lists both spellings, so I've changed it to the "ize" variant.